ui.frontend/src/main/components/App/App.js

```diff
@@ -35,11 +35,10 @@ import partialConfig from './config';
 import '../../site/main.scss';
 
 const App = () => {
-    const { storeView, graphqlEndpoint, graphqlMethod } = document.querySelector('body').dataset;
```

Comment:
Can you please add `storeView` to the config again? That's an important property used by the components and should not be removed.

Reply:
Ah you're right. I just put it back.

Reply:
Please add it back to the config as well.

Reply:
Yep should be fixed now.